src/Extension/ModuleHandler.php

```diff
@@ -41,7 +49,8 @@ class ModuleHandler implements ModuleHandlerInterface {
    * @see \Drupal\Core\DrupalKernel
    * @see \Drupal\Core\CoreServiceProvider
    */
-  public function __construct($root, array $module_list = array()) {
+  public function __construct(Drupal7 $drupal7, $root, array $module_list = array()) {
```

Comment:
Lets put drupal7 2nd, also need to update the doxygen, which is wrong anyway.

Scrutinizer reported that something calls this statically (createInstance or so) and misses the new argument.